101-azure-database-migration-service/azuredeploy.json

```diff
@@ -0,0 +1,594 @@
+
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
Done